sycl/doc/design/CommandGraph.md

```diff
@@ -109,6 +132,9 @@ data.
 Implementation of [UR command-buffers](#UR-command-buffer-experimental-feature)
 for each of the supported SYCL 2020 backends.
 
+This is currently only Level Zero but more sub-sections will be added here as
+we implement other backends.
+
 ### Level Zero
```

Comment:
I should document these drawbacks to our implementation highlighted by the maintainer
https://github.com/intel/llvm/pull/9992#discussion_r1246262690